src/common/constants.js

```diff
@@ -75,6 +76,7 @@ export const localized_link_url = Object.freeze({
     mt5: deriv_mt5_app_url,
     smart_trader: smarttrader_url,
     zoho: zoho_url,
+    'terms-and-conditions': terms_and_conditions_url,
```

Comment:
```suggestion
    terms-and-conditions: terms_and_conditions_url,
```
usually prettier will automatically remove quotes from object keys

Reply:
done